polaris/polaris/utils.py

```diff
@@ -268,7 +291,9 @@ def submit_stellar_deposit(transaction, multisig=False) -> bool:
             "Stellar transaction failed when submitted to horizon: "
             f"{transaction_result.result.results}"
         )
-
+    transaction.claimable_balance_id = (
+        get_balance_id(response) if transaction.claimable_balance_supported else None
+    )
```

Comment:
No need to optimize for one-liners, `claimable_balance_id` is already `None`
```suggestion
    if transaction.claimable_balance_supported:
        transaction.claimable_balance_id = get_balance_id(response)
```